docker-compose.yaml

```diff
@@ -11,6 +11,7 @@ services:
       test: ['CMD', 'curl', '-f', 'http://localhost:8080/auth/health']
     environment:
       - SPRING_PROFILES_ACTIVE=dev
+      - APPLICATION_AUTHENTICATION_UI_ALLOWLIST=0.0.0.0/0
```

Comment:
this allows access to `/auth/ui` so that client configuration can be modified / viewed